web-client/src/app/state/qAccess/q-access.service.ts

```diff
@@ -102,36 +102,44 @@ export class QAccessService implements OnDestroy {
     });
   }
 
-  async saveQuickAccess(address: string | undefined) {
+  async saveQuickAccess(
+    address: string | undefined,
+    promptForNickname: boolean = true
```

Comment:
why is the default for `promptForNickname` true?

Reply:
This is to ensure that the prompt does remain open in the case the user tries to submit an empty name , and are faced with an error.

Reply:
is there use cases where it can be false?

Reply:
No, there currently is not a case where it is false.

Reply:
I thought I'd do something similar for the invite code prompt, and cater for potential use cases. For now, there isn't a use case to have the parameter false. Do you advise that I remove it?

Reply:
no it's okay